library/ssl_tls13_keys.c

```diff
@@ -1141,7 +1141,7 @@ static int ssl_tls13_generate_early_key(mbedtls_ssl_context *ssl,
 
     ret = mbedtls_ssl_tls13_derive_early_secrets(
         hash_alg, handshake->tls13_master_secrets.early,
-        transcript, transcript_len, tls13_early_secrets);
+        transcript, transcript_len, &tls13_early_secrets);
```

Comment:
We can see, the keys generation functions will erase the temp secrets at the end of function, which make the secrets exist till the function ending(Taske transcript as one example). From a safe view, seems erasing them immediately is one safer way, but I have no idea why the original code is designed like this. easy to code?